docs/source/howto/run_pwbands.md

```diff
@@ -0,0 +1,133 @@
+(howto-workflows-pw-bands)=
+
+# Calculate a bands structure
+
+The `PwBandsWorkChain` is designed to compute electronic band structures for a given structure using Quantum ESPRESSO's `pw.x`. It automates the complete workflow: optional structure relaxation, SCF calculation, and bands calculation along high-symmetry k-points paths.
+
+|                     |                                                               |
+|---------------------|---------------------------------------------------------------|
+| Workchain class     | {class}`aiida_quantumespresso.workflows.pw.bands.PwBandsWorkChain` |
+| Workchain entry point | ``quantumespresso.pw.bands``                                |
+
+---
+
+## Minimal example: build and submit
+
+```python
+from aiida import orm, load_profile
+from aiida_quantumespresso.workflows.pw.bands import PwBandsWorkChain
+from aiida.engine import submit
+from ase.build import bulk
+
+load_profile()
+
+# Load your pw.x code and structure
+code = orm.load_code('pw@localhost')
+structure = orm.StructureData(ase=bulk('Si', 'diamond', 5.4))
+
+# Get a builder with sensible default parameters from a predefined protocol
+builder = PwBandsWorkChain.get_builder_from_protocol(
+    code=code,
+    structure=structure,
+    protocol="moderate",  # choose from: fast, moderate, precise
```

Comment:
Use the new alias of the protocols: 

* `moderate` -> `balanced`
* `precise` -> `stringent`